.github/workflows/ci.yml

```diff
@@ -25,15 +25,10 @@ jobs:
           tools: composer:v2
           coverage: none
 
-      - name: Install PHP 7 dependencies
-        run: composer update --prefer-dist --no-interaction --no-progress
-        if: "startsWith(matrix.php, '7.')"
-
-      - name: Install PHP 8 dependencies
+      - name: Install dependencies
         run: |
-          composer require "phpdocumentor/reflection-docblock:^5.2@dev" --no-interaction --no-update
-          composer update --prefer-dist --prefer-stable --no-interaction --no-progress --ignore-platform-req=php
-        if: "startsWith(matrix.php, '8.')"
+          composer update --prefer-dist --no-interaction --no-progress
```

Comment:
thank you for this cleanup. seems this was for very bleeding edge php 8.0 testing.